mitodl/unified-ecommerce-frontend
# Remove the API client that was copied from the Django app and used the published client instead

### What are the relevant tickets?

Closes mitodl/hq#6336

### Description (What does it do?)

What it says on the tin: removes the API client that we'd copied in from the Django app, and replaces it with the built client that's published to npm.

Also, this adds a `build` profile to the Docker env - you can use it to build the app, and then access it on port 7777.

### How can this be tested?

Rebuild and use the app - it should work as expected.

Run the Docker Compose environment with `--profile build`. Once everything has built and is running, access the app at `http://ue.odl.local:7777`. It should work as expected.
